library/core/class.request.php

```diff
@@ -897,6 +897,30 @@ protected function _setRequestArguments($paramsType, $paramsData = null) {
         $this->_RequestArguments[$paramsType] = $argumentData;
     }
 
+    /**
+     * Decode the environment's post depending on content type.
+     *
+     * @param array $post Usually the {@link $_POST} super-global.
+     * @param array $server Usually the {@link $_SERVER} super-global.
+     * @param array $inputFile Usually **php://input** for the raw input stream.
+     */
+    private function decodePost($post, $server, $inputFile) {
```

Comment:
You both have great points here. Since the method is private I'll go ahead and set the default, however @DaazKu is very correct about the general point.